lib/web/scripts/node-join/install.sh

```diff
@@ -777,9 +777,9 @@ fi
 if [[ ${TELEPORT_FORMAT} == "tarball" ]]; then
     # handle centos6 URL override
     if [[ ${DISTRO_TYPE} == "centos6" ]]; then
-        URL="https://get.gravitational.com/teleport-v${TELEPORT_VERSION}-${TELEPORT_BINARY_TYPE}-${TELEPORT_ARCH}-centos6-bin.tar.gz"
+        URL="https://get.gravitational.com/{{.packageName}}-v${TELEPORT_VERSION}-${TELEPORT_BINARY_TYPE}-${TELEPORT_ARCH}-centos6-bin.tar.gz"
```

Comment:
CentOS 6 is no longer supported by Teleport. I also think that the link now has `centos7` in the name. Should we update it?

Reply:
I think that should be a separate issue, don't you think?
Seems like we already support other `centos*` distro versions.
However you said that the link has `centos7` in the name, which is not present in the script.

Well, happy to tackle this one, but I think we should have a separate issue for it

Reply:
I missed the `DISTRO_TYPE` check. So, here is some background story. We used to support centos 6, and we had a special build for it. I think that with Teleport 9, we dropped support for CentOS 6, and then we changed the name of our tar to `centos7`. Later we decided that having two packages (one for CentOS 7 and one for everyone else) is too much work, and currently, we're releasing only one package that works everywhere.
Basically, this is dead code as CentOS 6 is not supported anymore, and we don't release packages with `centos6` in the name. I created an issue to not forget about it https://github.com/gravitational/teleport/issues/20813

Reply:
Thank you!